turbinia/client.py

```diff
@@ -235,9 +237,61 @@ def get_task_data(
 
     return results[0]
 
+  def format_task_detail(self, task, show_files=False):
+    """Formats a single task in detail.
+
+    Args:
+      task (dict): The task to format data for
+      show_files (bool): Whether we want to print out log file paths
+
+    Returns:
+      list: Formatted task data
+    """
+    report = []
+    saved_paths = task.get('saved_paths') or []
```

Comment:
task.get('saved_paths', []) instead?

Reply:
This is actually intentionally set this way, and the difference is that if `saved_paths` does have a value, but is set to `None`, then the returned value here will be `[]`.